client/src/Components/Pages/Category/index.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import girl from './images/girl.png';
+import teacher from './images/teacher.png';
+import company from './images/company.png';
+import ambassador from './images/ambassador.png';
+import './style.css';
+ const CategoryComponent = (props) => {
+    return (
+        <div className="signupCategory">
+             <NavLink to="/signup">
+               <div className="categoryElement">
+                   <img src={girl} alt="girl" />
+                   <h3>Girl Student</h3>
+                </div>
+             </NavLink>
```

Comment:
Will the other elements on this page (teacher, company, ambassador) link to other pages in the future?  There's nothing wrong with only linking the one element right now, I'm just asking so that I understand the intent of the category page better :)

Reply:
Sounds good, thanks!